plugins/modules/zos_started_task.py

```diff
@@ -0,0 +1,1639 @@
+          while starting it. If I(job_name) is not specified, then I(member_name) is used as job's name.
+        - When I(state) is C(displayed), C(modified), C(cancelled), C(stopped), or C(forced), I(job_name) is the
+          started task name.
+    required: false
+    type: str
+    aliases:
+        - job
+        - task
+        - task_name
+  keyword_parameters:
+    description:
+        - Any appropriate keyword parameter that you specify to override the corresponding parameter in the cataloged
+          procedure. The maximum length of each keyword=option pair is 66 characters. No individual value within this
+          field can be longer than 44 characters in length.
+        - Only applicable when I(state) is C(started), otherwise ignored.
+    required: false
+    type: dict
+  member_name:
+    description:
+        - Name of a member of a partitioned data set that contains the source JCL for the task to be started. The member
+          can be either a job or a cataloged procedure.
+        - Only applicable when I(state) is C(started), otherwise ignored.
+    required: false
+    type: str
+    aliases:
+        - member
+  parameters:
+    description:
+        - Program parameters passed to the started program.
+        - Only applicable when I(state) is C(started) or C(modified), otherwise ignored.
+    required: false
+    type: list
+    elements: str
+#   retry_force:
+#     description:
+#         - Indicates whether retry will be attempted on ABTERM(abnormal termination).
+#         - I(tcb_address) is mandatory to use I(retry_force).
+#         - Only applicable when I(state) is C(forced), otherwise ignored.
+#     required: false
+#     type: bool
+  reus_asid:
+    description:
+        - When I(reus_asid) is C(True) and REUSASID(YES) is specified in the DIAGxx parmlib member, a reusable ASID is assigned
+          to the address space created by the START command. If I(reus_asid) is not specified or REUSASID(NO) is specified in
+          DIAGxx, an ordinary ASID is assigned.
+        - Only applicable when I(state) is C(started), otherwise ignored.
+    required: false
+    type: bool
+  state:
+    description:
+        - I(state) should be the desired state of the started task after the module is executed.
+        - If I(state) is C(started) and the respective member is not present on the managed node, then error will be thrown with C(rc=1),
+          C(changed=false) and I(stderr) which contains error details.
+        - If I(state) is C(cancelled), C(modified), C(displayed), C(stopped) or C(forced) and the started task is not running on the managed node,
+          then error will be thrown with C(rc=1), C(changed=false) and I(stderr) contains error details.
+        - If I(state) is C(displayed) and the started task is running, then the module will return the started task details along with
+          C(changed=true).
```

Comment:
I think in this case changed should be False because other than issuing the command the module is not changing the managed node state.
```suggestion
          C(changed=false).
```